plugins/action/action.cpp

```diff
@@ -32,6 +32,11 @@ ActionResult Action::land() const
     return _impl->land();
 }
 
+ActionResult Action::goto_location(double latitude_deg, double longitude_deg, float altitude_amsl_m, float yaw_rad)
```

Comment:
Please expose yaw in degrees and convert it to rad internally.

Reply:
Done.